app/assets/stylesheets/components/hamburger.scss

```diff
@@ -2,28 +2,24 @@
 
 .hamburger {
```

Comment:
Hamburger menu (nav for mobile) is technically part of header, but its content is part of left sidebar so I considered this to be the right moment to update it as well.